kumascript/macros/XULAttr.ejs

```diff
@@ -34,12 +34,6 @@ var destOldest = lang + '/docs/XUL:' + attrName + ':' + $0;
 var destOld = lang + '/docs/XUL/' + attrName + '/' + $0;
 var destNew = lang + "/docs/" + newPath + "/" + $0;
 
-if (wiki.pageExists(destNew)) {
-    %><code id="<%= anch %>"><%-(web.link(wiki.uri(destNew), dispApi))%></code><%
-} else if (wiki.pageExists(destOld)) {
-    %><code id="<%= anch %>"><%-(web.link(wiki.uri(destOld), dispApi))%></code><%
-} else if (wiki.pageExists(destOldest)) {
-    %><code id="<%= anch %>"><%-(web.link(wiki.uri(destOldest), dispApi))%></code><%
-} else {
-    %><code id="<%= anch %>"><%-(web.link(wiki.uri(destNew), dispApi))%></code><%
-}%>
+%><code id="<%= anch %>"><%-(web.link(wiki.uri(destNew), dispApi))%></code>
+<code id="<%= anch %>"><%-(web.link(wiki.uri(destOld), dispApi))%></code>
+<code id="<%= anch %>"><%-(web.link(wiki.uri(destOldest), dispApi))%></code>
```

Comment:
These changes don't make sense. It should just be the first one.

Reply:
That's what I was thinking too. Just wanted to make sure with you before removing. Will update this shortly